docs/readmes/orc8r/deploy_build.md

```diff
@@ -79,6 +79,26 @@ COMPOSE_PROJECT_NAME=magmalte ${PUBLISH} -r ${REGISTRY} -i magmalte -v ${MAGMA_T
 ```
 
 ## Build and publish Helm charts
+*Only Choose one option below*
+### First option: Publish to artifactory
+
+We'll build the Orchestrator Helm charts, as well as publish them to an artifactory.
```

Comment:
Can this be any Helm chart repo, or does it need to be a [ChartMuseum](https://chartmuseum.com/) instance? If the latter, we should specify as much

Reply:
The scripts are working against an artifactory helm repo, it would not work with an helm ChartMuseum instance or other type.

Reply:
So is this script just something we'll use, special-purpose, for our magmacore repo? Or will it be generalize-able for anyone with a similar artifactory?